Extensions/BBText/bbtextruntimeobject-pixi-renderer.js

```diff
@@ -113,3 +113,11 @@ gdjs.BBTextRuntimeObjectPixiRenderer.prototype.getWidth = function() {
 gdjs.BBTextRuntimeObjectPixiRenderer.prototype.getHeight = function() {
   return this._pixiObject.height;
 };
+
+gdjs.BBTextRuntimeObjectPixiRenderer.prototype.setWidth = function(newWidth) {
+  this._pixiObject.width = newWidth;
```

Comment:
What is this doing to the pixi object? Is there something visible on screen?

Reply:
Text is stretched from the middle, not the best thing to do i agree, let starting by the first point, see my anwser bellow. 
Then we will be able to fix this.

Reply:
Can we just not implement this function (I mean, keep it empty) - because I don't think the text object was made to be stretched.

At least this is not a bug fix anymore but a new feature - and one to be considered very carefully because this could create confusion/complexity, especially with the fact that the text can be wrapped.